### ISSUE TITLE
[bug] keep talos install disk config even if not specified in omni template

### ISSUE THREAD

**Issue by seeekr** [siderolabs/omni-feedback#59]
### Is there an existing issue for this?

- [X] I have searched the existing issues _(I have actually not, frezbo asked me to make an issue here)_

### Current Behavior

Slack link: https://taloscommunity.slack.com/archives/C04D4PDAJT0/p1691958226088579

<img width="584" alt="CleanShot 2023-08-14 at 10 39 38@2x" src="https://github.com/siderolabs/omni-feedback/assets/302886/5dd797e9-c9eb-495a-956d-6c022773f77c">
<img width="578" alt="CleanShot 2023-08-14 at 10 40 03@2x" src="https://github.com/siderolabs/omni-feedback/assets/302886/bd9585d7-40dd-4570-acea-04b1db0754dc">

tl;dr error happens when omnictl template encounters a machine without install disk specified

### Expected Behavior

<img width="587" alt="CleanShot 2023-08-14 at 10 40 18@2x" src="https://github.com/siderolabs/omni-feedback/assets/302886/ad619aa9-4412-4c1e-8578-1d9f9ddc68ef">

tl;dr if installing via talos ISO was able to select (and wipe!) some default installation disk, then adding omni to the mix should not make talos "forget" it was able to do that. just reuse the same install disk that talos already picked earlier, and keep that as now-chosen install disk configuration.

### Steps To Reproduce

<img width="579" alt="CleanShot 2023-08-14 at 10 42 11@2x" src="https://github.com/siderolabs/omni-feedback/assets/302886/a5b29d2e-e3ac-4ae6-8d25-6159c6f1f5b9">

tl;dr just don't specify an install disk when adding to a cluster.

**Comment by Unix4ever** [comment]
This one should have been addressed in `v0.19.0`.
If you don't specify the install disk `Omni` will try to pick it automatically.
If Talos is installed (which includes booting Talos in maintenance mode from the raw image), Omni will generate the config in a way that Talos node won't complain about missing disk config.
